src/replication.c

```diff
@@ -2268,7 +2268,7 @@ void readSyncBulkPayload(connection *conn) {
                                       NULL);
 
                 disklessLoadDiscardTempDb(diskless_load_tempDb);
-                functionsLibCtxFree(temp_functions_lib_ctx);
+                functionsLibCtxFree(temp_functions_lib_ctx, 0);
```

Comment:
i will handle this diskless thing in other PR, we are doing async in temp db, and here with the temp function ctx, we should use async too

```
/* Discard tempDb, this can be slow (similar to FLUSHALL), but it's always async. */
void discardTempDb(serverDb *tempDb, void(callback)(dict *)) {
    int async = 1;

    /* Release temp DBs. */
    emptyDbStructure(tempDb, -1, async, callback);
    for (int i = 0; i < server.dbnum; i++) {
        kvstoreRelease(tempDb[i].keys);
        kvstoreRelease(tempDb[i].expires);
    }

    zfree(tempDb);
}
```